au4/src/au3wrap/CMakeLists.txt

```diff
@@ -377,24 +402,101 @@ set(AU3_SRC
     ${AU3_LIBRARIES}/lib-numeric-formats/ProjectTimeSignature.h
     ${AU3_LIBRARIES}/lib-numeric-formats/Beats.cpp
     ${AU3_LIBRARIES}/lib-numeric-formats/Beats.h
+    ${AU3_LIBRARIES}/lib-numeric-formats/NumericConverterFormats.cpp
+    ${AU3_LIBRARIES}/lib-numeric-formats/NumericConverterFormats.h
+    ${AU3_LIBRARIES}/lib-numeric-formats/NumericConverterRegistry.cpp
+    ${AU3_LIBRARIES}/lib-numeric-formats/NumericConverterRegistry.h
+    ${AU3_LIBRARIES}/lib-numeric-formats/ProjectNumericFormats.cpp
+    ${AU3_LIBRARIES}/lib-numeric-formats/ProjectNumericFormats.h
+    ${AU3_LIBRARIES}/lib-numeric-formats/NumericConverterFormatterContext.cpp
+    ${AU3_LIBRARIES}/lib-numeric-formats/NumericConverterFormatterContext.h
 
     # Requires wxApp, so we won't use it
     # ${AU3_LIBRARIES}/lib-wx-init/SettingsWX.cpp
     # ${AU3_LIBRARIES}/lib-wx-init/SettingsWX.h
+
+    ${AU3_LIBRARIES}/lib-effects/StatefulEffectBase.cpp
+    ${AU3_LIBRARIES}/lib-effects/StatefulEffectBase.h
+    ${AU3_LIBRARIES}/lib-effects/PerTrackEffect.cpp
+    ${AU3_LIBRARIES}/lib-effects/PerTrackEffect.h
+    ${AU3_LIBRARIES}/lib-effects/EffectBase.cpp
+    ${AU3_LIBRARIES}/lib-effects/EffectBase.h
+    ${AU3_LIBRARIES}/lib-effects/EffectPlugin.cpp
+    ${AU3_LIBRARIES}/lib-effects/EffectPlugin.h
+    ${AU3_LIBRARIES}/lib-effects/EffectOutputTracks.cpp
+    ${AU3_LIBRARIES}/lib-effects/EffectOutputTracks.h
+    ${AU3_LIBRARIES}/lib-effects/Effect.cpp
+    ${AU3_LIBRARIES}/lib-effects/Effect.h
+    ${AU3_LIBRARIES}/lib-effects/LoadEffects.cpp
+    ${AU3_LIBRARIES}/lib-effects/LoadEffects.h
+
+    ${AU3_LIBRARIES}/lib-audio-unit/AudioUnitEffectsModule.cpp
+    ${AU3_LIBRARIES}/lib-audio-unit/AudioUnitEffectsModule.h
+
+    ${AU3_LIBRARIES}/lib-command-parameters/ShuttleAutomation.cpp
+    ${AU3_LIBRARIES}/lib-command-parameters/ShuttleAutomation.h
+
+    ${AU3_LIBRARIES}/lib-vst3/AudacityVst3HostApplication.cpp
+    ${AU3_LIBRARIES}/lib-vst3/AudacityVst3HostApplication.h
+    ${AU3_LIBRARIES}/lib-vst3/VST3EffectBase.cpp
+    ${AU3_LIBRARIES}/lib-vst3/VST3EffectBase.h
+    ${AU3_LIBRARIES}/lib-vst3/VST3EffectsModule.cpp
+    ${AU3_LIBRARIES}/lib-vst3/VST3EffectsModule.h
+    ${AU3_LIBRARIES}/lib-vst3/VST3Instance.cpp
+    ${AU3_LIBRARIES}/lib-vst3/VST3Instance.h
+    ${AU3_LIBRARIES}/lib-vst3/VST3Utils.cpp
+    ${AU3_LIBRARIES}/lib-vst3/VST3Utils.h
+    ${AU3_LIBRARIES}/lib-vst3/VST3Wrapper.cpp
+    ${AU3_LIBRARIES}/lib-vst3/VST3Wrapper.h
+    ${AU3_LIBRARIES}/lib-vst3/internal/ConnectionProxy.cpp
+    ${AU3_LIBRARIES}/lib-vst3/internal/ConnectionProxy.h
+    ${AU3_LIBRARIES}/lib-vst3/memorystream.cpp
+    ${AU3_LIBRARIES}/lib-vst3/memorystream.h
```

Comment:
Question for the future, have you checked if there are any hard dependencies with some code from au3? Or can these sources be used independently by add them to some separate module?